sentence_transformers/sparse_encoder/evaluation/__init__.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+
+from sentence_transformers.sparse_encoder.evaluation.SparseBinaryClassificationEvaluator import (
+    SparseBinaryClassificationEvaluator,
+)
+from sentence_transformers.sparse_encoder.evaluation.SparseEmbeddingSimilarityEvaluator import (
+    SparseEmbeddingSimilarityEvaluator,
+)
+from sentence_transformers.sparse_encoder.evaluation.SparseInformationRetrievalEvaluator import (
+    SparseInformationRetrievalEvaluator,
+)
+from sentence_transformers.sparse_encoder.evaluation.SparseMSEEvaluator import (
+    SparseMSEEvaluator,
+)
+from sentence_transformers.sparse_encoder.evaluation.SparseMSEEvaluatorDataFrame import (
+    SparseMSEEvaluatorDataFrame,
+)
+from sentence_transformers.sparse_encoder.evaluation.SparseNanoBEIREvaluator import (
+    SparseNanoBEIREvaluator,
+)
+from sentence_transformers.sparse_encoder.evaluation.SparseRerankingEvaluator import (
+    SparseRerankingEvaluator,
+)
+from sentence_transformers.sparse_encoder.evaluation.SparseTranslationEvaluator import (
+    SparseTranslationEvaluator,
+)
+from sentence_transformers.sparse_encoder.evaluation.SparseTripletEvaluator import (
+    SparseTripletEvaluator,
+)
+
+__all__ = [
+    "SparseEmbeddingSimilarityEvaluator",
+    "SparseInformationRetrievalEvaluator",
+    "SparseBinaryClassificationEvaluator",
+    "SparseMSEEvaluator",
+    "SparseNanoBEIREvaluator",
+    "SparseTripletEvaluator",
+    "SparseTranslationEvaluator",
+    "SparseRerankingEvaluator",
+    "SparseMSEEvaluatorDataFrame",
+]
+
+
```

Comment:
We could experiment with an evaluator for hybrid (+ potentially reranker), ideally also with performances of individual components.